pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -10,7 +10,7 @@
 }: let
 
   name = "tmux-sessionizer";
-  version = "0.3.2";
+  version = "0.4.1";
```

Comment:
Not sure this is OK, since formally 0.4.1 is not released yet.

Reply:
0.4.0 doesn't build because of a version mismatch in Cargo.lock, so the options I see are either to set version to 0.4.0 but pin it to this commit https://github.com/jrmoulton/tmux-sessionizer/commit/79ab43a4087aa7e4e865cab6a181dfd24c6e7a90 (which would mean the package version and the output of `tms --version` don't match), or ask the project to tag a release

Reply:
What about `version = "unstable-2024-02-06";`?